### ISSUE TITLE
With CarryCapacity, game crashes when looking up chest in handbook

### ISSUE THREAD

**Issue by copygirl** [anegostudios/VintageStory-Issues#282]
Not quite sure how this happens. Perhaps something about adding [`BlockBehaviorCarryable`](https://github.com/copygirl/CarryCapacity/blob/v0.4.0-rc.1/src/Common/BlockBehaviorCarryable.cs) to the chest?
```
Running on 64 bit Windows with 32 GB RAM 
Version: v1.9.2 (Stable)5/10/2019 3:55:33 AM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.API.Common.CollectibleObject.GetHandbookInfo(ItemStack stack, ICoreClientAPI capi, ItemStack[] allStacks, Action`1 openDetailPageFor)
   at Vintagestory.API.Client.HandbookStacklistElement.GetPageText(ICoreClientAPI capi, ItemStack[] allStacks, Action`1 openDetailPageFor)
   at Vintagestory.GameContent.GuiDialogHandbook.initDetailGui()
   at Vintagestory.API.Client.GuiElementList.OnMouseUpOnElement(ICoreClientAPI api, MouseEvent args)
   at Vintagestory.API.Client.GuiElement.OnMouseUp(ICoreClientAPI api, MouseEvent args)
   at Vintagestory.API.Client.GuiComposer.OnMouseUp(MouseEvent mouse)
   at Vintagestory.API.Client.GuiDialog.OnMouseUp(MouseEvent args)
   at _r6tvNwMB2X4idaSEIw8v6E9ZkVg._BwnIvQVU3TEk5DqcOYysiarMKTb(MouseEvent )
   at _4tEiYgqX9Tbp0u6XQgZRQN2C00j._BwnIvQVU3TEk5DqcOYysiarMKTb(MouseEvent )
   at _Q4GkhL8fdNnbyajnFduEGWeDb1f._EbJegaica6JDJIgAIBSYY6dizQI(Object , MouseButtonEventArgs )
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wParam, IntPtr lParam)
   at OpenTK.Platform.Windows.Functions.DispatchMessage(MSG& msg)
   at OpenTK.Platform.Windows.WinGLNative.ProcessEvents()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _i2ESdD37oIQUDTNBET1OO6KKA5P._sEL4TL65ecQmjpf9cyH76MTBAuO(_PeiLuq7npShFRqWQoaqgQ2aAB7L , String[] )
   at _xzOEtcS097gqlSlz75yjjYiA28I._sEL4TL65ecQmjpf9cyH76MTBAuO(ThreadStart )
-------------------------------
```

**Comment by tyronx** [comment]
ok i think it know what it is. in blockbehavior.GetDrops() you return null. The handbook did not check for that.

**Comment by tyronx** [comment]
ok fixed for 1.9.3

**Comment by copygirl** [comment]
> in blockbehavior.GetDrops() you return null

Well, that's the default behavior that does it, I didn't touch it!
Thanks for the quick fix!
